src/modules/Chip/Chip.story.tsx

```diff
@@ -28,3 +30,25 @@ export const ChipIcon: React.FC<{
     />
   );
 };
+
+const DerivedChip = CloseableChip.extendStyles(() => ({
```

Comment:
Why so many changes to the chip? Is your branch rebased with master? Maybe that's the reason why you are apparently re-introducing old code